app/Http/Controllers/ReportsController.php

```diff
@@ -607,7 +607,7 @@ public function postCustom(Request $request)
             }
 
             if ($request->filled('by_company_id')) {
-                $assets->where('assets.company_id', $request->input('by_company_id'));
+                $assets->whereIn('assets.company_id', $request->input('by_company_id'));
```

Comment:
I'm not 100% sure how this would work, since the HTML would need to be passing an array not a singular variable.

Reply:
`WhereIn` passes in an array/all options selected. Where as `where` only grabs the first selected option only. I've tested this and it works 🤷‍♂️

Reply:
Right, but HTML -> server side expects a variable with a bracket to indicate that more than one value should be passed. `$foo` means the server will send one value for `$foo`, and in a multiselect, it will send whatever the last value was. `$foo[]` tells the server there will be several of a thing. The `whereIn()` makes sense here, I'm just not sure the HTML is set up to send the correct values. If you do a `dd()` on that field on submission, what do you see? Probably one value, not an array of values.